src/coreclr/tools/aot/Mono.Linker.Tests/TestCases/TestDatabase.cs

```diff
@@ -57,7 +58,7 @@ private static IEnumerable<TestCase> AllCases ()
 			return AllCases ().FirstOrDefault (c => c.Name == name);
 		}
 
-		private static IEnumerable<object[]> TestNamesBySuiteName (string suiteName)
+		private static IEnumerable<object[]> TestNamesBySuiteName ([CallerMemberName] string suiteName = "")
```

Comment:
Not feedback, just today I learned something new with this attribute. Looks nice :)

Reply:
Actually @sbomer used it first in the analyzer tests in linker, I just copied it.